tech.json

```diff
@@ -801,7 +801,20 @@
             "Used to fall on top of breakable bomb blocks or enemies.",
             "Performed by storing a spark while falling off a ledge, holding angle and crouching after storing a spark, or unmorphing."
           ],
-          "devNote": "This is not a momentum continuing action like canBlueSpaceJump"
+          "devNote": "This is not a momentum continuing action like canBlueSpaceJump",
+          "extensionTechs": [
+            {
+              "name": "canSpeedKeepChain",
+              "requires": [
+                "canSpeedKeep",
+                "canLateralMidAirMorph"
+              ],
+              "note": [
+                "Moving after obtaining a speedkeep by jumping into an air ball and then soft unmorph while holding an angle to maintain the speedkeep.",
```

Comment:
Its easier to control if you morph while falling, so "air ball" is confusing.
Move instead of Moving.

Reply:
Holding up or down is ok too.  That's why you can aim down and then press down again to morph.